ApplyToBecomeInternal/ApplyToBecome.Data/Services/AcademyConversionProjectRepository.cs

```diff
@@ -15,10 +16,12 @@ namespace ApplyToBecome.Data.Services
 	public class AcademyConversionProjectRepository : IAcademyConversionProjectRepository
 	{
 		private readonly HttpClient _httpClient;
+		private readonly ILegalRequirementsRepository _legalRequirementsRepository;
 
-		public AcademyConversionProjectRepository(IHttpClientFactory httpClientFactory)
+		public AcademyConversionProjectRepository(IHttpClientFactory httpClientFactory, ILegalRequirementsRepository legalRequirementsRepository)
 		{
 			_httpClient = httpClientFactory.CreateClient("TramsClient");
+			_legalRequirementsRepository = legalRequirementsRepository;
```

Comment:
This is not for now, but I think we'll have to remove this repository.
We should be able to get and set the legal requirements using the project repository.

Reply:
I agree, this is what I was referring to when messaging you earlier around the Legal Requirements seem to have been implemented differently to the other elements in the task list.